doc/source/release/1.25.dev0-notes.rst

```diff
@@ -0,0 +1,284 @@
+=============================
```

Comment:
Rebase error that this is included?

Reply:
yes, thanks. I generated it to see what it looks like and included it by mistake.